src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,364 @@
+---
+title: "Markdown indentation"
+tags:
+  - Basic style guide
+  - Style guide quick reference
+---
+
+import indentSnippetsInLists from 'images/indent-snippets-in-lists.png'
+
+import indentLeftJustified from 'images/indent-left-justified.png'
+
+import indentExtraLineFeed from 'images/indent-extra-line-feed.png'
+
+import topology1 from 'images/topology-1.png'
+
+import topology2 from 'images/topology-2.png'
+
+import childBulletList from 'images/child-bullet-list.png'
+
+import childNumberedList from 'images/child-numbered-list.png'
+
+import fourSpaceIndentChild from 'images/four-space-indent-child.png'
+
+import threeSpaceIndentChild from 'images/three-space-indent-child.png'
+
+import basicListIndentationLeft from 'images/basic-list-indentation-left.png'
+
+import basicListIndentationTwo from 'images/basic-list-indentation-two.png'
+
+
+When you're writing in Markdown, keep in mind that indentation can make your mark-up more readable, and it can also affect its readability when it's rendered in HTML.
+
+## First-level lists (parents) [#first-level]
+
+For first-level numbered and bulleted lists introduced by a left-justified introductory sentence, we follow the common Markdown standard: left-justify the first-level list items. At publication, the Markdown library will indent the first-level list items for you:
```

Comment:
```suggestion
For first-level lists introduced by a left-justified introductory sentence, we follow Markdown standard: left-justify the first-level list items. At publication, the Markdown library will indent the first-level list items for you:
```
Shortened the first clause because it's a mouthful. I figured if someone is savvy enough to seek out indentation tips, they'll assume 'lists' includes unordered and ordered lists. :) Also deleted common since 'standard' implies common anyway, and it shortens the sentence!